tools/azure-pipelines/jobs-template.yml

```diff
@@ -81,7 +81,7 @@ jobs:
   condition: and(succeeded(), not(eq(variables['MODE'], 'e2e')))
   pool: ${{parameters.test_pool_definition}}
   container: ${{parameters.container}}
-  timeoutInMinutes: 240
+  timeoutInMinutes: 60
```

Comment:
For the final PR this should be set to 2h or so (lower than the 4h we used before).

Reply:
Yes, it's part of a tmp commit.

Reply:
Why should we lower it? We did not make the tests run faster...

Reply:
The test didn't need 4h and they run quicker into a potential timeout this way. Just check the build times on recent builds (btw 4h is the default).